src/pages/ConnectionCallback.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <section>
+    <div class="">
+      <p class="title has-text-centered">
+        Connecting your {{service}} account...
+      </p>
+    </div>
+    <b-loading :active="true" />
+  </section>
+</template>
+
+<script>
+export default {
+  name: 'ConnectionCallback',
+  head: { title: { inner: 'Connecting' } },
+  mounted () {
+    this.setService()
+    this.connect()
+  },
+  data: () => ({ service: null }),
```

Comment:
The `setService` method is useless.
You can define it right in the `data`.
```js
data() {
    return { service: this.$route.params.connection }
}
```

Reply:
fixed